integration-tests/src/test/java/oracle/kubernetes/operator/ITOperator.java

```diff
@@ -30,38 +31,27 @@
 public class ITOperator extends BaseTest {
 
   // property file used to customize operator properties for operator inputs yaml
-  private static String op1YamlFile = "operator1.yaml";
-  private static String op2YamlFile = "operator2.yaml";
-  private static final String opForDelYamlFile1 = "operator_del1.yaml";
-  private static final String opForDelYamlFile2 = "operator_del2.yaml";
-  private static final String opForBackwardCompatibility = "operator_bc.yaml";
+  private static String opManagingdefaultAndtest1NSYamlFile =
+      "OperatorManagingdefaultAndtest1NS.yaml";
+  private static String opManagingtest2NSYamlFile = "OperatorManagingtest2NS.yaml";
+  private static final String opForBackwardCompatibility = "OperatorForBackwardCompatibility.yaml";
   private static final String opForRESTCertChain = "operator_chain.yaml";
 
-  // property file used to customize domain properties for domain inputs yaml
-  private static String domain1YamlFile = "domain1.yaml";
-  private static String domain2YamlFile = "domain2.yaml";
-  private static String domain3YamlFile = "domain3.yaml";
-  private static String domain4YamlFile = "domain4.yaml";
-  private static String domain5YamlFile = "domain5.yaml";
-  private static String domain6YamlFile = "domain6.yaml";
-  private static String domain7YamlFile = "domain7.yaml";
-  private static String domain8YamlFile = "domain8.yaml";
-  private static final String domain1ForDelValueYamlFile = "domain_del_1.yaml";
-  private static final String domain2ForDelValueYamlFile = "domain_del_2.yaml";
-  private static final String domain3ForDelValueYamlFile = "domain_del_3.yaml";
-  private static String domain9YamlFile = "domain9.yaml";
-  private static String domain10YamlFile = "domain10.yaml";
-  private static String domain11YamlFile = "domain11.yaml";
-  private static String domain12YamlFile = "domain12.yaml";
+  // file used to customize domain properties for domain, PV and LB inputs yaml
+  private static String domainOnPVUsingWLSTYamlFile = "DomainOnPVUsingWLST.yaml";
+  private static String domainOnPVUsingWDTYamlFile = "DomainOnPVUsingWDT.yaml";
+  private static String domainWithServerStartPolicyAsAdminOnlyYamlFile =
+      "DomainWithServerStartPolicyAsAdminOnly.yaml";
+  private static String domainWithStorageReclaimPolicyRecycleYamlFile =
+      "DomainWithStorageReclaimPolicyRecycle.yaml";
+  private static String domainWithDefaultValuesForSamplesYamlFile =
```

Comment:
Similar to my naming suggestion for domain files, maybe have var names like this be something like 'DOMAIN_UIDFile'.

Reply:
ok, changes made.